test/Microsoft.Azure.WebJobs.Host.EndToEndTests/WebJobs.Host.EndToEndTests.csproj

```diff
@@ -36,6 +36,15 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <LangVersion>default</LangVersion>
   </PropertyGroup>
+  <PropertyGroup>
+    <SignAssembly>true</SignAssembly>
+  </PropertyGroup>
+  <PropertyGroup>
+    <AssemblyOriginatorKeyFile>..\..\src\Common\PublicKey.snk</AssemblyOriginatorKeyFile>
```

Comment:
I access the internal setter for `HostMachineId` for testing, so I had to add `InternalsVisibleTo` to this project. And to do that, I needed it to be signed to match how we're doing it with the other test assemblies. I want to understand why it worked before, though. Suddenly changing the name to `HostMachineId` triggered 'not accessible' error messages. I'll figure it out before I commit.